sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div>
+    <div class="row">
+      <header-title-value
+        title="Submitted date"
+        :value="
+          headerDetails.submittedDate
+            ? dateOnlyLongString(headerDetails.submittedDate)
+            : '-'
+        "
+      />
+      <div
+        v-if="headerDetails.institutionName"
+        class="mx-2 vertical-divider"
+      ></div>
+      <header-title-value title="Institution"
+        ><template #value
+          ><span class="link-primary" @click="navigateToInstitutionProfile()">
+            {{ headerDetails.institutionName }}
+          </span>
+        </template></header-title-value
+      >
+      <div
+        v-if="headerDetails.locationName"
+        class="mx-2 vertical-divider"
+      ></div>
+      <header-title-value
+        v-if="headerDetails.locationName"
+        title="Location"
+        :value="headerDetails.locationName"
+      />
+    </div>
+    <div class="row mt-1" v-if="showReviewDetails">
+      <div
+        class="row pl-4"
+        v-if="
+          headerDetails.status ===
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent
+        "
+      >
+        <div class="mr-2">
+          <v-icon icon="fa:fas fa-times-circle" size="20" color="error" />
+        </div>
+        <header-title-value
+          :title="reviewLabel.assessedByLabel"
+          value="Student"
```

Comment:
Why value is `Student`, shouldn't it be student's actual name?

Reply:
There was a discussion with @JasonCTang  regarding this and this is what is expected currently. I have updated the details in the story as well. When the student declines the request, we only show the keyword "Student" and the date they declined the request.

Reply:
I'm confused with that solution. If the intention is to differentiate between ministry and student user, the feature status can be added to the page which will convey that.

Reply:
Yes, both options were presented.